tardis/plasma/base.py

```diff
@@ -258,10 +260,35 @@ def remove_hidden_properties(self, print_graph):
                 print_graph.remove_node(str(item.name))
         return print_graph
 
-class StandardPlasma(BasePlasma):
+    def to_hdf(self, path_or_buf, path='', close_hdf=True, filter=None):
```

Comment:
A complicated beast is an overstatement for these two simple classes, but the purpose there was to have some consistency in the different `to_hdf` signatures. This way, if we want to store part of plasma, and part of runner, and part of xyz, we only provide one Filter object to the first `simulation.to_hdf call`, instead of doing `simulation.to_hdf(plasma_properties, runner_properties, atom_properties, xyz_future_module_properties)`